src/containers/headerContainer.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from "react-redux";
+import Header from "../components/Header/header";
+import logout from "../actions/login/logout-action";
+import addEvent from "../actions/event/event-add-action";
+
+const mapStateToProps = state => {
+  const { user } = state;
+
+  return {
+    user,
+    hub: state.hubConnections.chatHub,
```

Comment:
why didn't you use desctruction for this row?

Reply:
I did not know how to do it. Thanks for help!